### ISSUE TITLE
Migration from old client does not enable notifications

### ISSUE THREAD

**Issue by Skirmisher** [vector-im/riotX-android#1723]
When upgrading from old Riot-Android to Element today, I had to manually check "Enable notifications for this session" under notification settings in the app. I gather that I would have been missing notifications had I not checked all the app settings (or if I hadn't understood the potentially confusing setting text and ignored it). If this requires intervention by the user for some reason, they should be prompted about it on first startup after migration.

**Comment by o0laxkilla0o** [comment]
I just checked my Element android client and can confirm this setting was disabled. I will reach out to my users which is a good mix of IOS and Android to see how their clients behaved after their migrations.

**Comment by o0laxkilla0o** [comment]
I have confirmed one IOS user has experienced this, after enabling notifications flowed as expected. It is late here tomorrow once I get more responses ill update the issue.

**Comment by hpdeifel** [comment]
I've observed this for three users, which is everyone that upgraded straight from riot to element. When I installed the beta a few weeks back, it correctly migrated the setting.

**Comment by RussellAult** [comment]
I can confirm this for myself and another user. We both upgraded with the Google Play change-over, and neither of us were receiving notifications after the upgrade.

If it makes a difference we're using a private HomeServer, and when I ran the "Troubleshoot Notifications" function, in addition to notifications being disabled for the session, it was complaining that the FCM token wasn't registered to the HomeServer. Hitting the "register" button solved that problem, and enabling session notifications got notifications flowing again. After this, enabling session notifications was enough for things to start working again for the other user as well.

I can't say for sure, but it also looks like it's not importing the advanced notification settings either (e.g. I swear I had encrypted messages sent to a group set to "noisy").
